src/appengine/handlers/configuration.py

```diff
@@ -116,6 +116,8 @@ def post(self):
     contact_string = request.get('contact_string')
     documentation_url = request.get('documentation_url')
     github_credentials = request.get('github_credentials')
+    oss_fuzz_robot_github_personal_access_token = request.get(
+        'oss_fuzz_robot_github_personal_access_token')
```

Comment:
you'll also need to edit https://github.com/google/clusterfuzz/blob/481c78b28fb2d9a401a6f3fe302746a8f6110954/src/appengine/private/components/configuration/configuration.html so that we can edit this in UI. 

You can check the UI locally by doing `python butler.py run_server`.